src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATE_HISTORY = [];
```

Comment:
The variable `STATE_HISTORY` should follow camelCase naming convention. Consider renaming it to `stateHistory`.